app/Gists/Gistlog.php

```diff
@@ -111,13 +112,17 @@ public function getPreview()
         return substr($body, 0, strpos($body, ' ', 200));
     }
 
-    private function renderMarkdown()
+    private function renderMarkdown($content = null)
     {
```

Comment:
Yeah @adamwathan totally what I was thinking... took the words right out of my mouth... you just got to it before I did :) 

Also [this](https://sourcemaking.com/refactoring/replace-conditional-with-polymorphism) has been up in my browser since you said "Replace Conditional with Polymorphism" at Laracon. Great use case!